app/views/api/v2/auth_source/main.json.rabl

```diff
@@ -0,0 +1,5 @@
+object @auth_source
+
+extends "api/v2/auth_source/base"
+
+attributes :onthefly_register, :usergroup_sync, :tls, :server_type, :created_at, :updated_at
```

Comment:
Yeah, it's caused by STI. It can be confusing.
From the list above I think it's only the timestamps.